src/tests/ov_helpers/ov_snippets_models/src/subgraph_mlp_seq.cpp

```diff
@@ -192,12 +182,7 @@ std::shared_ptr<ov::Model> MLPSeqQuantizedTypeRelaxedFunction::initReference() c
         subgraph_params.push_back(B);
         subgraph_nodes.push_back(B_const_trans);
 
-        current = decomposed_fq(current,
-                                ov::element::u8,
-                                onData.inputLowValues[0],
-                                onData.inputHighValues[0],
-                                0.00346764503f);
-        current = std::make_shared<ov::snippets::op::ConvertSaturation>(current, ov::element::u8);
+        current = FakeQuantizeFunction::getDecomposedFakeQuantizeOps(current, ov::element::u8, 1.f, 20.f, 13.4211f);
```

Comment:
Minor: can we preserve previous values?
```suggestion
        current = FakeQuantizeFunction::getDecomposedFakeQuantizeOps(current, ov::element::u8, onData.inputLowValues[0], onData.inputHighValues[0], 0.00346764503f);
```